src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -160,8 +200,8 @@ void GcsServer::DoStart(const GcsInitData &gcs_init_data) {
   // Init gcs health check manager.
   InitGcsHealthCheckManager(gcs_init_data);
 
-  // Init KV Manager
-  InitKVManager();
+  // Init KV service.
+  InitKVService();
```

Comment:
Should we remove this? It's already called in `GcsServer::Start`

Reply:
One is KV Manager and one is KV service